invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -57,7 +67,7 @@ function InVideoQuizXBlock(runtime, element) {
     }
 
     function getDimensions() {
-        var position = $('.tc-wrapper', video).position().top;
+        var position = $('.tc-wrapper', video).position();
         var height = $('.tc-wrapper', video).css('height');
         var width = $('.tc-wrapper', video).css('width');
```

Comment:
Just FYI. In js this
```
var foo;
var baz;
var bar;
```
is bad practice.
This 
```
var foo,
    baz,
    bar;
```
is better. Just for the future 👻